msgcntr/messageforums-app/src/webapp/js/forum.js

```diff
@@ -877,6 +877,8 @@ $(document).ready(function(){
 
       const rubricGrading = document.getElementsByTagName("sakai-rubric-grading").item(0);
       rubricGrading && rubricGrading.save();
+      $(rubricGrading).find('input').each(function() {
```

Comment:
rubricGrading && rubricGrading.querySelectorAll("input").forEach(i => i.prop("disabled", true));

Reply:
What are you disabling here? Is it the score override boxes?

Reply:
If it is the comment boxes, you should probably be more specific
"input.fine-tune-points"